build_runner/lib/src/package_graph/apply_builders.dart

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:build/build.dart';
+
+import '../generate/phase.dart';
+import 'package_graph.dart';
+
+bool _alwaysTrue(_) => true;
+
+bool Function(PackageNode) _dependencyFilter(String dependency) =>
```

Comment:
Done keeping it private so we don't need to expose more symbols than we already are. I also made the `filter` field private since it shouldn't necessarily be useful elsewhere and we want to focus on the named constructors.